haystack/document_store/sql.py

```diff
@@ -16,42 +16,43 @@
 class ORMBase(Base):
     __abstract__ = True
 
-    id = Column(String, default=lambda: str(uuid4()), primary_key=True)
+    id = Column(String(100), default=lambda: str(uuid4()), primary_key=True)
     created = Column(DateTime, server_default=func.now())
     updated = Column(DateTime, server_default=func.now(), server_onupdate=func.now())
 
 
 class DocumentORM(ORMBase):
     __tablename__ = "document"
 
-    text = Column(String, nullable=False)
-    index = Column(String, nullable=False)
-    vector_id = Column(String, unique=True, nullable=True)
+    text = Column(Text, nullable=False)
+    index = Column(String(100), nullable=False)
+    vector_id = Column(String(100), unique=True, nullable=True)
 
     # speeds up queries for get_documents_by_vector_ids() by having a single query that returns joined metadata
     meta = relationship("MetaORM", backref="Document", lazy="joined")
 
+
 class MetaORM(ORMBase):
     __tablename__ = "meta"
 
-    name = Column(String, index=True)
-    value = Column(String, index=True)
-    document_id = Column(String, ForeignKey("document.id", ondelete="CASCADE"), nullable=False)
+    name = Column(String(100), index=True)
+    value = Column(String(100), index=True)
```

Comment:
Title of the doc is stored as value of `meta.name` key and there is possibility that it can breach 100 char limit. I feel `Text` would be good choice for `value`.

Reply:
Well spotted @lalitpagaria! I switched the `value` field to `Text`.